torchvision/models/detection/transform.py

```diff
@@ -86,10 +85,9 @@ def __init__(self, min_size, max_size, image_mean, image_std, size_divisible=32,
         self.fixed_size = fixed_size
 
     def forward(self,
-                images,       # type: List[Tensor]
-                targets=None  # type: Optional[List[Dict[str, Tensor]]]
-                ):
-        # type: (...) -> Tuple[ImageList, Optional[List[Dict[str, Tensor]]]]
+                images: List[Tensor],
+                targets: Optional[List[Dict[str, Tensor]]] = None
+                ) -> Tuple[ImageList, Optional[List[Dict[str, Tensor]]]]:
```

Comment:
Hey @datumbox !
We don't need to hack this anyomore!
Unoin is supported by Torchscript #4360 

We can use `Union[ImageList, Optional[List[Dict[str, Tensor]]]`

I'm currently working to use this and possibly finish #4220

Reply:
Aah found it https://github.com/pytorch/vision/pull/4220#discussion_r679218040 yeah 
I commented on wrong code.

Reply:
I'm not sure this is related to the problem you describe. This is not a workaround, could you clarify if your comment is still valid here?

Reply:
Yeah its' invalid. My bad

Reply:
no worries :) thanks for checking though